docs/src/main/asciidoc/_cloud/Getting_Started.adoc

```diff
@@ -0,0 +1,147 @@
+There are many options available for running OpenShift. This guide will mention two.
+
+##### Code-ready-containers
+If you wish to run a local development environment, then https://developers.redhat.com/products/openshift-local/overview[CRC] is likely to be a good starting point.
+
+##### Developer sandbox
+https://www.redhat.com/en/technologies/cloud-computing/openshift/try-it[The Red Hat OpenShift Developer Sandbox is a hosted environment], and is free to use for development purposes.
+
+This guide will focus on usage of the hosted developer sandbox environment. The sandbox should be provisioned from the page above before proceeding further.
+
+### OpenShift
+In order to use OpenShift effectively, in addition to a running instance, you will also need the OpenShift client tools (`oc` command). This may be obtained from your package manager or downloaded from your OpenShift sandbox (in your control panel, clicking on the question mark icon at the top right of the console presents a drop-down menu where `Command line tools` takes you to the latest version of `oc`).
+
+Once you have downloaded and installed/unpacked the client, you can authenticate to your running sandbox OpenShift instance by using the `Copy Login Command` from the upper right menu in the sandbox user-interface (your username is displayed there.)
+
+Example:
+```
+$ oc login --token=sha256~b-3av_0Lp8wmJu0G3WXzdfds34454mkvWjRRESd_jHTA --server=https://api.sandbox.x8i5.p1.openshiftapps.com:6443
+```
+
+We'll be using the `someuser-dev` project for this example:
+
+```
+$ oc project someuser-dev
+Already on project "someuser-dev" on server "https://api.sandbox.x8i5.p1.openshiftapps.com:6443"
+```
+
+In the OpenShift sandbox, it is not possible to create a new project but, in case you are running your own OpenShift, the command to create a new project is:
+```
+oc new-project <project name>
+```
+
+### Helm Charts
+https://helm.sh/[Helm] provides an easy way to manage and share applications on Kubernetes and OpenShift. We will use it to deploy our example applications to OpenShift.
+
+##### Install Helm and Helm Charts
+See the instructions for installing Helm https://helm.sh/docs/intro/install/[here]. Helm is installed on your local system as a command, which we will use to install the WildFly Charts. Once Helm is installed, we can proceed to use it to install the Helm Charts in our OpenShift instance:
+
+```
+$ helm repo add wildfly https://docs.wildfly.org/wildfly-charts/
+```
+
+For more information on WildFly Helm Charts, see this https://www.wildfly.org/news/2021/05/05/helm-charts-for-wildfly/[blog post] and the https://docs.wildfly.org/wildfly-charts/[WildFly Helm Charts documentation]
+
+## Example: To-do Quickstart
+The first example we will build and deploy to OpenShift is the https://github.com/wildfly/quickstart/tree/main/todo-backend[todo-backend quickstart]. This quickstart provides documentation for installing via Helm, so only a brief version is detailed below.
```

Comment:
Would be good to add a link to the OpenShift instructions from the todo-backend quickstart, e.g.,

"documentation" could link to https://github.com/wildfly/quickstart/tree/main/todo-backend#run-the-backend-on-openshift